hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -1014,6 +1014,62 @@ ContainerCommandResponseProto handleWriteChunk(
     return getWriteChunkResponseSuccess(request, blockDataProto);
   }
 
+  /**
+   * Handle Write Chunk operation for closed container. Calls ChunkManager to process the request.
+   *
+   */
+  public void writeChunkForClosedContainer(ChunkInfo chunkInfo, BlockID blockID,
+                                           ChunkBuffer data, KeyValueContainer kvContainer)
+      throws IOException {
+
+    try {
+      Preconditions.checkNotNull(chunkInfo);
+      Preconditions.checkNotNull(data);
+      long writeChunkStartTime = Time.monotonicNowNanos();
+      checkContainerClose(kvContainer);
+
+      DispatcherContext dispatcherContext = DispatcherContext.getHandleWriteChunk();
+      chunkManager.writeChunk(kvContainer, blockID, chunkInfo, data,
+          dispatcherContext);
+
+      // Increment write stats for WriteChunk after write.
+      metrics.incContainerBytesStats(Type.WriteChunk, chunkInfo.getLen());
```

Comment:
Write new metrics, this should be a separate processing that we need to track from the normal write chunk path.